dev/com.ibm.ws.jaxws.ejb_fat/publish/servers/com.ibm.ws.jaxws.ejb.fat.ejbinwarservice/server.xml

```diff
@@ -18,6 +18,8 @@
   	<javaPermission className="java.io.FilePermission" name="ALL FILES" actions="read"/>
   	<javaPermission className="java.lang.reflect.ReflectPermission" name="suppressAccessChecks"/>
 
+
+    <javaPermission className="org.osgi.framework.AdminPermission" name="*" actions="*"/>
```

Comment:
I've added this and all the following for java 2 sec failures in our tests framework that are going to require some complicated doPrivs that will need to be added after the next update to CXF is done.